ui/src/components/connections/ConnectionDetails.tsx

```diff
@@ -122,29 +274,64 @@ export function ConnectionDetails() {
               <>
                 <Avatar className="avatar-color-cyan" icon={<IconCreditCardRefresh />} size={48} />
 
-                <Typography.Text strong>No credentials issued</Typography.Text>
+                <Typography.Text strong>
+                  No {credentialType !== "all" && credentialType} credentials issued
```

Comment:
If you'd have used a string template literal instead, you'd have seen that this concatenation can yield the message:

> No false credentials issued

<img width="1346" alt="image" src="https://user-images.githubusercontent.com/15896177/229841706-49f5d95d-b547-48a3-9675-71135278d1aa.png">